barretenberg/cpp/src/barretenberg/eccvm/eccvm_composer.test.cpp

```diff
@@ -14,26 +14,31 @@
 #include "barretenberg/sumcheck/sumcheck_round.hpp"
 
 using namespace bb;
-using G1 = bb::g1;
-using Fr = bb::fr;
 
 class ECCVMComposerTests : public ::testing::Test {
   protected:
-    // TODO(640): The Standard Honk on Grumpkin test suite fails unless the SRS is initialized for every test.
     void SetUp() override { srs::init_grumpkin_crs_factory("../srs_db/grumpkin"); };
 };
 namespace {
 auto& engine = numeric::get_debug_randomness();
 }
+
+/**
+ * @brief Adds operations in BN254 to the op_queue and then constructs and ECCVM circuit from the op_queue.
+ *
+ * @param engine
+ * @return ECCVMCircuitBuilder
+ */
 ECCVMCircuitBuilder generate_circuit(numeric::RNG* engine = nullptr)
 {
-    std::shared_ptr<ECCOpQueue> op_queue = std::make_shared<ECCOpQueue>();
+    using Curve = curve::BN254;
+    using G1 = Curve::Element;
+    using Fr = Curve::ScalarField;
 
-    auto generators = G1::derive_generators("test generators", 3);
-
-    typename G1::element a = generators[0];
-    typename G1::element b = generators[1];
-    typename G1::element c = generators[2];
+    std::shared_ptr<ECCOpQueue> op_queue = std::make_shared<ECCOpQueue>();
```

Comment:
Why did you change this?

Reply:
To make it uniform to the recursive verifier but can revert